core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/actions/resource/CreateLabel.java

```diff
@@ -0,0 +1,80 @@
+package de.valtech.aecu.core.groovy.console.bindings.actions.resource;
+
+import de.valtech.aecu.api.service.AecuException;
+import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
+import org.apache.sling.api.resource.PersistenceException;
+import org.apache.sling.api.resource.Resource;
+
+import javax.annotation.Nonnull;
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+/**
+ *@author Ian Smets
+ *@author Wietse Vandeput
+**/
+public class CreateLabel implements Action {
+    private final String language;
+    private final String key;
+    private final String value;
+    String PROPERTY_SLING_MESSAGE = "sling:message";
+    String MIXIN_SLING_MESSAGE = "sling:Message";
+    String PROPERTY_SLING_MESSAGE_ENTRY = "sling:MessageEntry";
+    String MIXIN_TYPE_MIX_LANGUAGE = "mix:language";
+    String PROPERTY_JCR_LANGUAGE = "jcr:language";
+
+    public CreateLabel(String language, String key, String value){
+        this.language = language;
+        this.key = key;
+        this.value = value;
+    }
+
+    @Override
+    public String doAction(@Nonnull Resource resource) throws PersistenceException, AecuException {
+        Session session = resource.getResourceResolver().adaptTo(Session.class);
+        try {
+            Node i18nNode = session.getNode(resource.getPath());
+            Node node = getLanguageNode(i18nNode, language);
+            if (node != null) {
+                try {
```

Comment:
3 nested try-blocks, all catching `RepositoryException` and throwing a `RuntimeException`
=> reduce to 1 try-block for everything, unless you really want different catch-clauses for specific errors.  just don't nest them like this
=> wrap the `RepositoryException` in a `PersistenceException` and add a new, clear message